Chart.yaml

```diff
@@ -10,4 +10,4 @@ version: 0.1.2
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.6.0-rc.40"
+appVersion: "0.6.0"
```

Comment:
Armory Cloud support has not been released yet in a stable version. For using these helm charts with the rebranding the only supported version is `0.7.0-rc.1`